src/github.com/getlantern/peerscanner/mem_20150120_085312.309949401.prof

```diff
@@ -0,0 +1,601 @@
+heap profile: 16: 330528 [2713: 37329872] @ heap/1048576
```

Comment:
Thanks, definitely not. I accidentally committed a bunch of files in the peerscanner directory. I added *.prof, *.gz, and *.pem to .gitignore